src/sql/platform/connection/common/connectionStatusManager.ts

```diff
@@ -49,6 +50,21 @@ export class ConnectionStatusManager {
 		return !!this.findConnection(id);
 	}
 
+	public changeConnectionUri(newId: string, oldId: string) {
```

Comment:
lets be consistent and rename the arguments to newUri and oldUri

Reply:
I was being consistent with the rest of the file, but if we want it to be consistent with the other calls, sure thing.